benchmarks/benchmarks.rb

```diff
@@ -245,7 +245,9 @@ def positional_splat_keyword_and_double_splat_args(a, *args, b:, **kwargs)
 
   # Print headers based on the first benchmark_json
   if i.zero?
-    benchmark_headers = benchmark_json.map do |benchmark_gem|
+    benchmark_headers = benchmark_json.
```

Comment:
We repeat this `sort_by` code later on line 262. Wondering if it'd make sense to put it into a `sorted_benchmark_json` variable before this `if` so we can use it later too?